src/ResourceTemplate/RelativeResourceTemplate.php

```diff
@@ -272,12 +272,15 @@ public function match($path)
         return $collapsedBindings;
     }
 
-    private function matchException($path, $reason)
+    /**
+     * @param string|null $path
```

Comment:
Missing one param here? `$reason`?

Reply:
I only added it because I couldn't typehint it. But since this is a private method, I've gone back and fixed it so the typehint is valid.